lib/plugins/pluginEndpoints

```diff
@@ -0,0 +1,117 @@
+const { default: axios } = require('axios')
+const util = require('util')
+const request = require('request').defaults({ encoding: null })
+const post = util.promisify(request.post)
+const get = util.promisify(request.get)
+
+const config = require('../config')
+
+module.exports = function (req, res) {
+
+    const name = req.query.name;
+    const endpoint = req.query.endpoint;
+    const [pluginUrl, category] = findPlugin(name);
+
+
+    if (pluginUrl === null) {
+        return res.status(404).send('Plugin not found');
+    }
+    
+
+
+    switch (endpoint) {
+        case 'status': {
+            getStatus(pluginUrl, res);
+            break;
+        }
+        case 'evaluate': {
+            const data = decodeURIComponent(req.query.data); 
+            getEvaluate(pluginUrl, data, category, res);
+            break;
+        }
+        case 'run': {
+            const data = decodeURIComponent(req.query.data); 
+            getRun(pluginUrl, data, category, res);
+            break;
+        }
+        default: res.status(400).send('Error with request');
```

Comment:
I think this could be clearer: e.g. this plugin type is not known, try evaluate, status, run, or call instead